src/item.cpp

```diff
@@ -2237,12 +2264,16 @@ bool item::is_auxiliary_gunmod() const
 
 int item::get_storage() const
 {
-    const auto t = find_armor_data();
+    float pockets = 1;
+    auto t = find_armor_data();
     if( t == nullptr ) {
         return 0;
     }
+    if (pocketed == true){
+        pockets = (1.5); //Test value
+    }
     // it_armor::storage is unsigned char
-    return static_cast<int>( static_cast<unsigned int>( t->storage ) );
+    return (static_cast<int> (static_cast<unsigned int>( t->storage * pockets) ) );
```

Comment:
I suggest you add your modification outside of the casts.

The casts are there because casting an `unsigned char` to `signed int` involves two (internal) casts: first to `signed char`, than to `signed int`. The first cast to `signed char` can change the number to become negative: `unsigned char x = 255; signed char y = x;` - y will be -1. Casting to `unsigned int` first, avoids that problem.

Why did you remove the `const` from `t`, you don't change it, so why does it have to be non-const?

```C++
if( pockets ) {
...
```

Also, you don't need that many brackets: `x = 1.5;` is totally fine, the added brackets in the return statement is similar useless.